include/zephyr/bluetooth/audio/pbp.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+
+/**
+ * @brief Public Broadcast Profile (PBP)
+ *
+ * @defgroup bt_pbp Public Broadcast Profile (PBP)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * [Experimental] Users should note that the APIs can change
+ * as a part of ongoing development.
+ */
+
+#include <zephyr/sys/util.h>
+
+#include <zephyr/bluetooth/audio/audio.h>
+
+/** PBP Roles */
+enum bt_pbp_role {
+	BT_PBP_ROLE_PBS = BIT(0), /*< Public Broadcast Source */
+	BT_PBP_ROLE_PBK = BIT(1), /*< Public Broadcast Sink */
+	BT_PBP_ROLE_PBA = BIT(2), /*< Public Broadcast Assistant */
+};
```

Comment:
I do not see much value in having these. Their only purpose seems to restrict users from using functions that does not affect behavior (i.e. setting or parsing advertising data). I suggest we remove these.